packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionFootNoteTitle/JurisdictionFootNoteTitle.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { Text } from '@deriv/quill-design';
+import { THooks } from '../../../../../types';
+import { MarketTypeDetails } from '../../../constants';
+
+type TJurisdictionFootNoteTitle = {
+    marketType: keyof typeof MarketTypeDetails;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+};
+
+const JurisdictionFootNoteTitle: FC<TJurisdictionFootNoteTitle> = ({ marketType, selectedJurisdiction }) => {
+    let footnoteText: string | undefined;
+
+    switch (selectedJurisdiction) {
+        case 'svg':
+            footnoteText = `Add your Deriv MT5 ${MarketTypeDetails[marketType].title} account under Deriv (SVG) LLC (company no. 273 LLC 2020).`;
+            break;
+        case 'bvi':
+            footnoteText = `Add your Deriv MT5 ${MarketTypeDetails[marketType].title} account under Deriv (BVI) Ltd, regulated by the British Virgin Islands Financial Services Commission (License no. SIBA/L/18/1114).`;
+            break;
+        case 'labuan':
+            footnoteText = `Add your Deriv MT5 ${MarketTypeDetails[marketType].title} STP account under Deriv (FX) Ltd regulated by Labuan Financial Services Authority (License no. MB/18/0024).`;
+            break;
+        case 'vanuatu':
+            footnoteText = `Add your Deriv MT5 ${MarketTypeDetails[marketType].title} account under Deriv (V) Ltd, regulated by the Vanuatu Financial Services Commission.`;
+            break;
+        default:
+            footnoteText = undefined;
+            break;
+    }
```

Comment:
In my opinion its better to use object literal and using keys to return footerText based on the selectedJurisdiction

Reply:
I think it is a better approach we can tweak base on other requirements.
For now it is okay.